### ISSUE TITLE
Make Window Re-Sizeable via Drag&Drop

### ISSUE THREAD

**Issue by Rhinogradentia** [warpdotdev/Warp#4641]
### Describe the solution you'd like?

Obviously, it should be possible to configure the terminal's window size in the settings menu. But it would be great if this could be done using the mouse/cursor via drag & drop, as it is possible in almost every other window in a GUI. 

### How important is this feature to you?

2

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Hi @Rhinogradentia, I'm unable to reproduce this issue of resizing windows with mouse on Mac and Ubuntu.

https://github.com/warpdotdev/Warp/assets/16809145/6192b4ee-359d-4a81-8bf9-852155173b65

Which OS are you on? Are you using any particular window manager or tiling manager tools? Are you on the latest version of Warp?

**Comment by Rhinogradentia** [comment]
Hi @dannyneira, thank you for your answer. I suspect it has something to do with my setup. I did install it under WSL2/Ubuntu, and the Window opens as a GUI within Windows 11. I used the starship bashrc configuration, but I already deactivated it again. This changed nothing. I do not use any particular window or tiling manager (just the build-in windows tiling function). It should be the latest version of warp - just did an update.

https://github.com/warpdotdev/Warp/assets/12270382/31ee2cd7-2b61-4f04-b871-a9828fda98f4

When double-clicking the window top it just expands over the whole screen. So I have 2 modi - full screen or the size you see in the video. Maybe I have something wrong in the settings?
